python/python/glide/async_commands/core.py

```diff
@@ -33,7 +33,7 @@
 
 class ConditionalChange(Enum):
     """
-    A condition to the "SET" and "ZADD" commands.
+    A condition to the `SET`, `ZADD` and `GEOADD` commands.
     - ONLY_IF_EXISTS - Only update key / elements that already exist. Equivalent to `XX` in the Redis API
```

Comment:
'Redis', redis.io and other possible trademarks.
I dont think we should continue to introduce these into our code.
Instead, I propose to use RESP3 i.e. 'Equivalent to `XX` in the RESP protocol.
Please consult with @asafpamzn ( you can setup a call for all of us)